app/views/bookings/show.html.erb

```diff
@@ -24,13 +24,13 @@
   <% else %>
     <!-- For lent -->
     <%= simple_form_for @booking, method: :patch do |f| %>
-      <div class="container shadow pb-3 py-2">
+      <div class="pb-3 py-2">
         <%= f.input :message, label: false, placeholder: "Message pour Capy #{@booking.user.username}", input_html:{ autofocus: true } %>
-        <div class="form-row d-flex date-fields-container">
+        <div class="form-row d-flex justify-content-around date-fields-container mt-2 mb-4">
           <% if @booking.status == "pending" %>
-            <%= f.collection_radio_buttons :status, [[:confirmed, 'Accepter'], [:declined, 'Refuser']], :first, :last %>
+            <%= f.collection_radio_buttons :status, [[:confirmed, ' Accepter'], [:declined, 'Refuser']], :first, :last, item_wrapper_class: 'form-check-inline mr-4', wrapper_html: { class: 'mt-3' } %>
```

Comment:
Why there is a space before `Accepter`? in `' Accepter'`

Reply:
it's fix !